src/api/supportRequest.ts

```diff
@@ -186,10 +187,14 @@ supportRequestRoutes.patch('/getSpecific', async (req, res) => {
     return;
   }
   try {
+    const newHash = genHash();
     await SupportRequest.createQueryBuilder('supportRequest')
       .update()
       .set({
         status: SupportRequestStatus.InProgress,
+        movedToInProgressAt: new Date(),
+        syncHash: newHash,
+        supportName,
```

Comment:
Why were these added? 🤔 I may have forgotten if we talked about it, but I'm curious why they weren't there before.

Reply:
`/supportRequest/getSpecific` was not updating those fields in the database and was causing errors when I tried to assign items to people. My guess is since `/getSpecific` was never really used before (old code only used `/getNext`), this part of the code was never tested.